### ISSUE TITLE
UDF does not work with Python installed from Microsoft Store

### ISSUE THREAD

**Issue by FA-DavidKun** [xlwings/xlwings#2371]
#### OS (e.g. Windows 10 or macOS Sierra)
Windows 11

#### Versions of xlwings, Excel and Python (e.g. 0.11.8, Office 365, Python 3.7)
xlwings: Version: 0.30.13; Excel: Version 2311 (build 17029.20068 Click-to-Run); Python 3.11 (installed from the Microsoft Store)

#### Describe your issue (incl. Traceback!)
UDFs don't work (error Could not activate python COM server) when using Python installed from the Microsoft store

#### Include a minimal code sample to reproduce the issue (and attach a sample workbook if required!)
Error occurs when clicking the Import Functions button on the xlwings tab of the Ribbon.

Similar but distinct from the solved issue #895

**Comment by fzumstein** [comment]
Yeah I believe that the Windows store version of Python has some limitations, so best to avoid it.
